workflow/rocoto/rocoto.py

```diff
@@ -144,8 +145,11 @@ def _create_innermost_task(task_dict: Dict[str, Any]) -> List[str]:
         strings.append(f'\t<partition>{partition}</partition>\n')
     strings.append(f'\t<walltime>{walltime}</walltime>\n')
     strings.append(f'\t<nodes>{nodes}:ppn={ppn}:tpp={threads}</nodes>\n')
-    if memory is not None:
-        strings.append(f'\t<memory>{memory}</memory>\n')
+
+    pw_csp = os.environ.get('PW_CSP')
+    if pw_csp not in ['aws', 'azure', 'google']:
```

Comment:
rocoto.py need not know if this is PW_CSP.
Please outline the issue and we can discuss.